<launch>
  <arg name="input_cloud" default="/filtered_cloud" />
  <arg name="output_cloud" default="/robot_filtered_cloud" />
  <arg name="filter_config_file" default="robot_self_filter.yaml" />

  <node pkg="robot_self_filter"
        type="self_filter" clear_params="true"
        name="self_filter" respawn="true" output="screen">
      <remap from="cloud_in" to="$(arg input_cloud)" />
      <remap from="cloud_out" to="$(arg output_cloud)" />
      <rosparam command="load" file="$(find robot_self_filter)/config/$(arg filter_config_file)" />
      <param name="use_rgb" value="false" />
      <param name="keep_organized" value="true"/>
      <param name="subsample_value" type="double" value="0.0"/>
      </node>
</launch>
